Sources/Scene/Assets/AssetsViewController.swift

```diff
@@ -241,9 +274,25 @@ extension AssetsViewController: PHPhotoLibraryChangeObserver {
                 self.fetchResult = changes.fetchResultAfterChanges
                 self.collectionView.reloadData()
             }
-
+            
             // No matter if we have incremental changes or not, sync the selections
             self.syncSelections(self.store.assets)
         }
     }
 }
+
+extension AssetsViewController: AutorizationStatusHeaderViewDelegate {
+    func didTapManageButton(for status: PHAuthorizationStatus) {
+        switch status {
+        case .denied:
+            self.handleDeniedAccess()
```

Comment:
controller is not shown if no access (full or limited), so `.denied` is never executed

Reply:
removed handling of `.denied` access